src/main/scala/abalone/Pos.scala

```diff
@@ -121,22 +282,84 @@ object Pos {
   val C1 = new Pos(2)
   val D1 = new Pos(3)
   val E1 = new Pos(4)
-  val F1 = new Pos(5)
-  val G1 = new Pos(6)
-  val H1 = new Pos(7)
-  val I1 = new Pos(8)
-
-  // backwards for 2nd rank
-  val I2 = new Pos(9)
-  val H2 = new Pos(10)
-  val G2 = new Pos(11)
-  val F2 = new Pos(12)
+  // val F1 = new Pos(5) //
+  // val G1 = new Pos(6) //
+  // val H1 = new Pos(7) //
+  // val I1 = new Pos(8) //
+  val A2 = new Pos(9) // means the 9th square of our square grid is represented by A2 (index starting bottom left)
+  val B2 = new Pos(10)
+  val C2 = new Pos(11)
+  val D2 = new Pos(12)
   val E2 = new Pos(13)
-  val D2 = new Pos(14)
-  val C2 = new Pos(15)
-  val B2 = new Pos(16)
-  val A2 = new Pos(17)
-
+  val F2 = new Pos(14)
+  // val G2 = new Pos(15)
+  // val H2 = new Pos(16)
+  // val I2 = new Pos(17)
+  val A3 = new Pos(18)
+  val B3 = new Pos(19)
+  val C3 = new Pos(20)
+  val D3 = new Pos(21)
+  val E3 = new Pos(22)
+  val F3 = new Pos(23)
+  val G3 = new Pos(24)
+  // val H3 = new Pos(25) //
+  // val I3 = new Pos(26) //
+  val A4 = new Pos(27)
+  val B4 = new Pos(28)
+  val C4 = new Pos(29)
+  val D4 = new Pos(30)
+  val E4 = new Pos(31)
+  val F4 = new Pos(32)
+  val G4 = new Pos(33)
+  val H4 = new Pos(34)
+  // val I4 = new Pos(35) //
+  val A5 = new Pos(36)
+  val B5 = new Pos(37)
+  val C5 = new Pos(38)
+  val D5 = new Pos(39)
+  val E5 = new Pos(40)
+  val F5 = new Pos(41)
+  val G5 = new Pos(42)
+  val H5 = new Pos(43)
+  val I5 = new Pos(44)
+  // val A6 = new Pos(45) //
+  val B6 = new Pos(46)
+  val C6 = new Pos(47)
+  val D6 = new Pos(48)
+  val E6 = new Pos(49)
+  val F6 = new Pos(50)
+  val G6 = new Pos(51)
+  val H6 = new Pos(52)
+  val I6 = new Pos(53)
+  // val A7 = new Pos(54) //
+  // val B7 = new Pos(55) //
+  val C7 = new Pos(56)
+  val D7 = new Pos(57)
+  val E7 = new Pos(58)
+  val F7 = new Pos(59)
+  val G7 = new Pos(60)
+  val H7 = new Pos(61)
+  val I7 = new Pos(62)
+  // val A8 = new Pos(63) //
+  // val B8 = new Pos(64) //
+  // val C8 = new Pos(65) //
+  val D8 = new Pos(66)
+  val E8 = new Pos(67)
+  val F8 = new Pos(68)
+  val G8 = new Pos(69)
+  val H8 = new Pos(70)
+  val I8 = new Pos(71)
+  // val A9 = new Pos(72) //
+  // val B9 = new Pos(73) //
+  // val C9 = new Pos(74) //
+  // val D9 = new Pos(75) //
+  val E9 = new Pos(76)
+  val F9 = new Pos(77)
+  val G9 = new Pos(78)
+  val H9 = new Pos(79)
+  val I9 = new Pos(80)
+
+  // @TODO: does not generate the hexagon coordinates, just the 9x9 square - do we want it to generate the hexagon only ?
   val all: List[Pos] = (0 to (File.all.size * Rank.all.size) - 1).map(new Pos(_)).toList
```

Comment:
yes, this needs to generate only the valid `Pos` for the hexagon, so it needs changing